build.gradle

```diff
@@ -2,7 +2,7 @@ plugins {
   id 'java'
   id 'application'
   id 'com.github.johnrengelman.shadow' version '2.0.4'
-  id "com.diffplug.gradle.spotless" version '3.14.0'
+  id "com.diffplug.gradle.spotless" version '3.15.0'
```

Comment:
The [old version](https://plugins.gradle.org/plugin/com.diffplug.gradle.spotless/3.14.0) is still in gradle so this _should_ have succeeded.

Reply:
Opened up #90 to increase gradle's HTTP connection timeouts

Reply:
We don't really benefit from the spotless `3.15.0` release, closing this PR. Thanks!

Reply:
Gotcha, my fault, I thought this was a workaround to 3.14 failing to download.

Reply:
In a way, it was sort of a workaround and my attempt to secretly bump the version at the same time :)